src/website/views/components/avatar.pug

```diff
@@ -102,6 +103,8 @@ ul#a-tabs--colors.a-tabs
       <div class="a-avatar -green">AA</div>
       <div class="a-avatar -yellow">AA</div>
       <div class="a-avatar -orange">AA</div>
+      <div class="a-avatar -secondary">AA</div>
```

Comment:
@davidojedacl @mattnickles shouldn't we remove secondary example from white backgrounds and just leave it in dark backgrounds as we don't want to encourage users to use it in this way?

Reply:
Agree, let's remove.

Reply:
done